ui/editor/src/view.ts

```diff
@@ -317,6 +317,12 @@ function inputs(ctrl: EditorCtrl, fen: string): VNode | undefined {
         attrs: { readonly: true, spellcheck: 'false', value: ctrl.makeEditorUrl(fen, ctrl.bottomColor()) },
       }),
     ]),
+    h('p', [
+      h('strong.name', 'IMAGE'),
```

Comment:
Would be good to align the labels

Reply:
Instead of showing the raw URL would be nicer to have the same design as under game analysis https://private-user-images.githubusercontent.com/271432/299477251-448fa1ad-9afe-4226-a441-c55ab9e14ba1.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MDYxOTQ4NDcsIm5iZiI6MTcwNjE5NDU0NywicGF0aCI6Ii8yNzE0MzIvMjk5NDc3MjUxLTQ0OGZhMWFkLTlhZmUtNDIyNi1hNDQxLWM1NWFiOWUxNGJhMS5wbmc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjQwMTI1JTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI0MDEyNVQxNDU1NDdaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT1mNTk0MGMyOWEyYzU5N2MzNDRhY2VlYTY3MTg1ZTBlNzgyZDBlYTk5MGM2MjMxN2RiNTBlOWJiM2FhYmQyZWIwJlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCZhY3Rvcl9pZD0wJmtleV9pZD0wJnJlcG9faWQ9MCJ9.SAXTFzSXFBKA81rWnDpHL8nX9HrhIYmoNfYAqEOojTE